source/write-scripts.txt

```diff
@@ -197,15 +202,162 @@ the following calls within the |mdb-shell| are equivalent:
 .. code-block:: javascript
    :copyable: false
 
-   load("scripts/connect-and-insert.js")
-   load("/data/db/scripts/connect-and-insert.js")
+   load( "scripts/connect-and-insert.js" )
+   load( "/data/db/scripts/connect-and-insert.js" )
 
 .. note::
 
    There is no search path for the :method:`load()` method. If the target
    script is not in the current working directory or the full specified
    path, the |mdb-shell| cannot access the file.
 
+Execute a Script From the Command Line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use :binary:`~bin.mongosh` to execute a script from the command
+line without entering the :binary:`~bin.mongosh` console. 
+
+Use the ``--file`` or ``-f`` parameters to specify the filename.
+
+.. code-block::
+
+   mongosh --port 27500 --file addMovies.js 
+
+.. tip::
+
+   If you pass a filename to :binary:`~bin.mongosh` without using the
+   parameter flags the connection may fail if there are other command
+   line arguments.
+   
+   To pass filenames always use ``--file`` or ``-f``.
+
+You may also need to specify connection information in addition to the
+``--file`` or ``-f`` parameters. 
+
+.. example::
+
+   The following example creates scripts and runs them from the command
+   line. 
+   
+   - ``loadMovies.js``, uses :method:`~db.collection.insertMany()` to a
+     update a local MongodDB instance.
+   - ``queryMovies.js`` uses :method:`db.collection.find()` to verify
+     the update.
+
+   #. Copy this script and save it as ``loadMovies.js``.
+
+      .. code-block:: javascript
+         :emphasize-lines: 1
+
+         db = connect( 'mongodb://localhost/films' );
+
+         db.movies.insertMany( [
+            {
+               title: 'Titanic',
+               year: 1997,
+               genres: [ 'Drama', 'Romance' ]
+            },
+            {
+               title: 'Spirited Away',
+               year: 2001,
+               genres: [ 'Animation', 'Adventure', 'Family' ]
+            },
+            {
+               title: 'Casablanca',
+               genres: [ 'Drama', 'Romance', 'War' ]
+            }
+         ] )
+
+      .. tip::
+   
+         Verify the connection string in the highlighted line. If your
+         MongoDB instance is not running on ``localhost:27017``, you
+         must edit the connection string. 
+
+         .. code-block:: javascript
+            :emphasize-lines: 1
+
+            db = connect( 'mongodb://localhost:27500/films' );
+
+      This connection string is for the:
+
+      - films database
+      - located on localhost
+      - listening on port 27500
+
+   #. Copy this script and save it as ``queryMovies.js``.
+
+      .. code-block:: javascript
+
+         db = connect( 'mongodb://localhost/films' );
+         printjson( db.movies.find( {} ) ); 
+
+   #. Run the scripts from the command line.
+
+      .. code-block:: javascript
+
+         mongosh --file loadMovies.js -f queryMovies.js
```

Comment:
Is this supposed to be run with `--nodb`? Otherwise this will fail if no server is running on localhost:27017. I think you generally want to use `--nodb` if you assign `db` at the beginning of your script, or alternatively use something like `db = db.getSiblingDB('films')` instead of the `connect()` call.

(I think in practice the latter might have more benefits, since it means that the script is independent from your specific deployment.)